### PR TITLE
User module

### PR DESCRIPTION
**Pull request by namithachandran**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```
Comment (eshaiju) [inline app/models/user.rb:4]: Need to add confirmable to work confirmation

**Comment by namithachandran** [comment]
Hi Shaiju,

This is only the first task. I am working on the confirmation task and will
make necessary changes.

Regards,
Namitha Chandran
ROR Intern
Ruby Software

**Comment by eshaiju** [comment]
Since you added confirmation related fields into migration, I was expected
to add confirmable.

**Comment by namithachandran** [comment]
Hi Shaiju,

It was suggested by JK to enable the fields and i am working on the
confirmation task now. Do i need to change?

**Pull request merged**